packages/selleo-design-core/src/Button.tsx

```diff
@@ -1,54 +1,63 @@
-      "text-brand-primary-500 border border-solid border-brand-primary-500 hover:bg-brand-primary-100 focus-visible:bg-brand-primary-100 focus-visible:outline focus-visible:outline-1 focus-visible:outline-offset-2 focus-visible:outline-brand-primary-500":
-        variant === "primaryOutlined",
-      "flex-col items-start text-neutral-500 dark:text-neutral-300 group hover:text-brand-primary-500 dark:hover:text-brand-primary-500 focus-visible:outline-0 after:w-[30%] after:h-[2px] after:bg-neutral-500 dark:after:bg-neutral-300 after:transition-all after:duration-300 after:rounded after:mt-[2px] hover:after:bg-brand-primary-500 dark:hover:after:bg-brand-primary-500 hover:after:w-full focus-visible:after:bg-brand-primary-500 focus-visible:after:w-full disabled:after:bg-brand-primary-400":
-        variant === "link",
-      "py-1 px-2 text-xs": size === "small",
-      "py-1 px-3 text-sm": size === "normal",
-    }
+      'text-white bg-brand-primary-500 hover:bg-brand-primary-600 focus-visible:bg-brand-primary-600 focus-visible:outline focus-visible:outline-1 focus-visible:outline-brand-primary-600 focus-visible:outline-offset-2':
+        variant === 'primary',
+      'text-brand-primary-500 border border-solid border-brand-primary-500 hover:bg-brand-primary-100 focus-visible:bg-brand-primary-100 focus-visible:outline focus-visible:outline-1 focus-visible:outline-offset-2 focus-visible:outline-brand-primary-500':
+        variant === 'primaryOutlined',
+      'flex-col items-start text-neutral-500 dark:text-neutral-300 group hover:text-brand-primary-500 dark:hover:text-brand-primary-500 focus-visible:outline-0 after:w-[30%] after:h-[2px] after:bg-neutral-500 dark:after:bg-neutral-300 after:transition-all after:duration-300 after:rounded after:mt-[2px] hover:after:bg-brand-primary-500 dark:hover:after:bg-brand-primary-500 hover:after:w-full focus-visible:after:bg-brand-primary-500 focus-visible:after:w-full disabled:after:bg-brand-primary-400':
+        variant === 'link',
+      'py-1 px-2 text-xs': size === 'small',
+      'py-1 px-3 text-sm': size === 'normal',
+    },
+    classes
   );
 
-  const contentClasses = cx("flex items-center justify-center px-0.5 rounded", {
-    "group-focus-visible:bg-brand-primary-500 group-focus-visible:text-white":
-      variant === "link",
-    "gap-0.5": size === "small",
-    "gap-1": size === "normal",
-  });
+  const contentClasses = cx(
+    'flex items-center justify-center px-0.5 rounded w-100% text-center',
+    {
+      'group-focus-visible:bg-brand-primary-500 group-focus-visible:text-white':
+        variant === 'link',
+      'gap-0.5': size === 'small',
+      'gap-1': size === 'normal',
+    }
+  );
 
-  const iconSize = size === "small" ? 16 : 24;
+  const iconSize = size === 'small' ? 16 : 24;
 
   return (
     <button class={buttonClasses} {...rest}>
       <div class={contentClasses}>
         {IconStart && <IconStart width={iconSize} height={iconSize} />}
-        Text
+        {children}
```

Comment:
it does break on Button page due to the missing content, can you try to find a way to fix it ? :D

Reply:
Yes, if there is no text content provided, default text appear
{children || "Text"}
Have you that kind of solution on mind? :D

Reply:
if it works like that sure :D but we have seen issues with the `children` props so I'm asking :D

Reply:
Now in a button component "children" are rendering as expected :)